lib/src/models/field.dart

```diff
@@ -70,6 +71,28 @@ class Field extends Equatable {
   List<Object> get props => [name, type, isMultivalued];
 }
 
+class DroppableField extends Field {
```

Comment:
Hey @kishorenc 
Since only fields can be updated as of now, I've created `DroppableField` class to be used in the update function.

Reply:
@happy-san I think having an `UpdateSchema` class as detailed in the [issue](https://github.com/typesense/typesense-dart/issues/118) is better because we can add other updatable fields to it in future. By using a `List< DroppableField>` that flexibility is lost. With `UpdateSchema`, the update operation will look like this:

```
final updateSchema = UpdateSchema(
  {
    Field('company_category', Type.string),
    Field('num_employees', drop: true)
  }
);
await client.collection('companies').update(updateSchema);
```

This also means that we need to add a `drop` property to the field. We debated the semantics of adding this, but any alternative way of representing this became way too verbose.

Reply:
Makes sense, I'll update this accordingly.